### ISSUE TITLE
Twisted Conch setup.py gives its name as "Conch" instead of "Twisted Conch"

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#2391]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)| @exarkun reported|
|-|-|
|Trac ID|trac#2391|
|Type|defect|
|Created|2007-01-11 17:42:44Z|
|Branch|https://github.com/twisted/twisted/tree/conch-setup-name-2391|

<details><summary>Searchable metadata</summary>

```
trac-id__2391 2391
type__defect defect
reporter__exarkun exarkun
priority__normal normal
milestone__Twisted_10_0 Twisted-10.0
branch__branches_conch_setup_name_2391 branches/conch-setup-name-2391
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1168537364000000 1168537364000000
changetime__1261338328000000 1261338328000000
version__None None
owner__ 
cc__thijs
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun commented|
|-|-|

(In [27302]) Branching to 'conch-setup-name-2391'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [27303]) Change the name to "Twisted Conch"

refs [#2391](https://github.com/twisted/twisted/issues/2391)

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_4"></a>|@thijstriemstra commented|
|-|-|

Seems ready to apply to me.

**Comment by twisted-trac** [comment]
|[<img alt="jesstess's avatar" src="https://avatars.githubusercontent.com/u/188336?s=50" width="50" height="50">](https://github.com/jesstess)<a name="note_5"></a>|@jesstess set status to `closed`|
|-|-|

(In [27840]) Merge conch-setup-name-2391

Authors: exarkun
Reviewers: thijs
Fixes: [#2391](https://github.com/twisted/twisted/issues/2391)

In setup.py, call the project "Twisted Conch" instead of "Conch".

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
